src/ThemeProvider.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes, Component} from 'react'
```

Comment:
Kent we are the only two people on Earth who do this. Go team lowercase-with-dashes!